File: app/123go/app/src/main/java/com/example/thien/a123go/models/Counter.java
package com.example.thien.a123go.models;

/**
 * Created by thien on 5/1/2017.
 */

public class Counter {
    public final String name;
    public final String address;
    public final String phone;
    public final String description;
    public final String image;

    public Counter(String name,String address, String phone, String description, String image){
        this.name = name;
        this.address = address;
        this.phone = phone;
        this.description = description;
        this.image = image;
    }
}

File: backend/sources/app_quan_an/app/Models/Counter.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Counter extends Model
{
    protected $table = "counter";

    /**
     * @var array
     */
    protected $fillable = [
        'owner_id',
        'type',
        'name',
        'address',
        'phone',
        'description',
        'image',
    ];
}
